docs/cloud/Generic-cloud.md

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Generic Cloud (cloud-init)'
+---
+# AlmaLinux OS Generic Cloud (cloud-init) image
+
+The Generic Cloud image is a general purpose virtual machine image that
+contains the [cloud-init](https://cloud-init.io/) package. During boot,
+cloud-init will take configuration options from cloud metadata and
+initialize a system accordingly to it. This may include network
+configuration, user's SSH key pair injection, attaching store devices,
+and so on.
+
+
+## Download and verification
+
+The AlmaLinux OS Generic Cloud image is available from our
+[mirrors](https://mirrors.almalinux.org/) and from the main AlmaLinux OS
+repository [repo.almalinux.org/almalinux/8/cloud/x86_64/images/](https://repo.almalinux.org/almalinux/8/cloud/x86_64/images/)
+(an aarch64 version is coming soon).
+
+
+### Import the AlmaLinux OS PGP public key
+
+In order to verify a downloaded image you need to import the AlmaLinux OS PGP
+public key first.
+
+If you are using an AlmaLinux OS-powered system, you may skip the command
+below because you already have the key stored in the
+`/etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux` file. Otherwise, download the key:
+
+
+```
+$ curl -s https://repo.almalinux.org/almalinux/RPM-GPG-KEY-AlmaLinux -o RPM-GPG-KEY-AlmaLinux
+```
+
+print the key fingerprint:
+
+```
+$ gpg --with-subkey-fingerprints RPM-GPG-KEY-AlmaLinux
+gpg: WARNING: no command supplied.  Trying to guess what you mean ...
+pub   rsa4096 2021-01-12 [C] [expires: 2024-01-12]
+      5E9B8F5617B5066CE92057C3488FCF7C3ABB34F8
+uid           AlmaLinux <packager@almalinux.org>
+sub   rsa3072 2021-01-12 [S] [expires: 2024-01-12]
+```
+
+the fingerprint is `5E9B8F5617B5066CE92057C3488FCF7C3ABB34F8`. If you see a
+different fingerprint, it means you downloaded a compromised file. Please,
+[let us know](mailto:security@almalinux.org) a mirror from which you
+downloaded the file, remove the file and retry the download from another
+mirror.
+
+next you need to import the key:
+
+```
+$ gpg --import RPM-GPG-KEY-AlmaLinux
+gpg: key 488FCF7C3ABB34F8: public key "AlmaLinux <packager@almalinux.org>" imported
+gpg: Total number processed: 1
+gpg:               imported: 1
+```
+
+
+### Download and verify the image
+
+Download checksum file and its signature first:
+
+```
```

Comment:
@ezamriy checksum files are missing. is the URLs correct or files to be uploaded? May be https://repo.almalinux.org/almalinux/8/cloud/x86_64/images/

Reply:
Nice catch, thanks! Fixed.